src/components/Notifications/NotificationFeed/NotificationFeed.js

```diff
@@ -508,12 +525,27 @@ class NotificationFeed extends React.Component {
   }
 
   renderDeleteAllButton = () => {
-    return null
-    // Enable this once the batch delete endpoint is ready
+    let title = 'Delete all notifications?'
+    if (this.props.enableFetchAllNotificationFeedAcrossProjects) {
+      title = `Delete all notifications for ${this.props.selectedProjectName}?`
+    } else {
+      return
+      //Enable this button for alpha alerts only temporarily
```

Comment:
I don't think this comment makes sense to have here anymore now that the button is being rendered. 

One change you could do here is instead of having an empty return in the else block, at the top of the function you can do 
`if (!this.props.enableFetchAllNotificationFeedAcrossProjects) {
return 
}

// render button logic after...
`